### PR TITLE
AI product description: Promo dialog

### LINKED ISSUE #9255: Add a product description AI generation promo dialog to My store screen

### PR DESCRIPTION
**Pull request by 0nko**
Fixes #9255, a subtask of #9214.

This PR adds a new promo dialog for the AI product description to the My store screen. The dialog buttons don't do anything. It is currently hidden behind a feature flag.

image

**To test:**

1. Start the app in debug mode
2. Log in, if logged out
3. Notice the AI product description dialog is shown when the My store screen is loaded

### CODE REVIEW

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/mystore/AIProductDescriptionDialogFragment.kt [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        if (isTabletLandscape()) {
+            requireDialog().window!!.setLayout(
+                (DisplayUtils.getWindowPixelWidth(requireContext()) * TABLET_LANDSCAPE_WIDTH_RATIO).toInt(),
+                (DisplayUtils.getWindowPixelHeight(requireContext()) * TABLET_LANDSCAPE_HEIGHT_RATIO).toInt()
+            )
+        }
+    }
+
+    private fun isTabletLandscape() = (DisplayUtils.isTablet(context) || DisplayUtils.isXLargeTablet(context)) &&
+        DisplayUtils.isLandscape(context)
```
Comment (JorgeMucientes) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/mystore/AIProductDescriptionDialogFragment.kt:77]: Nice touch, stretching the dialog bounds for tablets 👍🏼

**Review by JorgeMucientes — APPROVED** [review]

**Comment by JorgeMucientes** [comment]
UI and code look good! Well done @0nko! PR ready to merge once `MyStoreViewModelTest.kt` compilation issues are fixed.

**Comment by jaclync** [comment]
👋 Saw this PR from the Kiwi backlog board, just wanted to share that there's a discussion pecCkj-Cr-p2#comment-347 on changing this modal to use the What's New announcement modal since the modal of this design was currently used for JITM.

**Comment by 0nko** [comment]
Thanks for the heads up, @jaclync! I'll move on to another task until a decision is made.

**Pull request merged**